src/Paket.Core/Versioning/FrameworkHandling.fs

```diff
@@ -513,7 +513,7 @@ module FrameworkDetection =
                 | "netcore10" -> Some (DotNetCore DotNetCoreVersion.V1_0)
                 | "netcore11" -> Some (DotNetCore DotNetCoreVersion.V1_1)
                 | "netcore20" -> Some (DotNetCore DotNetCoreVersion.V2_0)
-                | v when v.StartsWith "netstandard" -> Some(DotNetStandard DotNetStandardVersion.V1_6)
+                //| v when v.StartsWith "netstandard" -> Some(DotNetStandard DotNetStandardVersion.V1_6)
```

Comment:
not sure I want to do that it was already wrong for the "netstandard" case and is basically wrong when new ones are added